sys/termios_bsd.go

```diff
@@ -0,0 +1,134 @@
+// +build darwin dragonfly freebsd netbsd openbsd
+
+// Copyright 2015 go-termios Author. All Rights Reserved.
+// https://github.com/go-termios/termios
+// Author: John Lenton <chipaca@github.com>
+
+package sys
+
+import (
+	"os"
+	"time"
+
+	"golang.org/x/sys/unix"
+)
+
+// Apparently missing from sys/unix
+const (
+	// taken from sys/file.h
+	FREAD  = 1
+	FWRITE = 2
+
+	// taken from termios.h
+	TCOOFF = 1
+	TCOON  = 2
+	TCIOFF = 3
+	TCION  = 4
+)
+
```

Comment:
Only consonants in the const block below and setFlag are used.

Please stripe the file and (termios_linux.go) to include only these consonants (see setFlag for why it should also be removed). Since these codes are forked instead of vendored, developers of Elvish will be maintaining these code, and we should maintain as little code as possible. :)